### PR TITLE
Format the elapsed time without sub-second precision.

### PR DESCRIPTION
**Pull request by petermattis**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -37,6 +38,16 @@ var (
 	flagStdErr   = flag.Bool("stderr", false, "output failures to STDERR instead of to a temp file")
 )
 
+// Format a duration without the sub-second precision.
+func fmtDuration(d time.Duration) string {
+	s := d.String()
```
Comment (tamird) [inline main.go]: This should properly handle the case of `d` being less than one second.

> // String returns a string representing the duration in the form "72h3m0.5s".
> // Leading zero units are omitted.  As a special case, durations less than one
> // second format use a smaller unit (milli-, micro-, or nanoseconds) to ensure
> // that the leading digit is non-zero.  The zero duration formats as 0,
> // with no unit.
> func (d Duration) String() string {

Comment (petermattis) [inline main.go]: Sure. Done.

**Comment by tamird** [comment]
LGTM

**Comment by tamird** [comment]
Actually, a simpler version is possible: `(time.Duration(d.Seconds())*time.Second).String()`

**Comment by petermattis** [comment]
I like that better, though I've made it `roundToSeconds`.

**Comment by tamird** [comment]
:+1:

**Comment by BramGruneir** [comment]
Cool.  I'll update the version used for the nightly stress tests in AWS as well.

**Pull request merged**